res/css/views/messages/_MLocationBody.scss

```diff
@@ -23,6 +23,9 @@ limitations under the License.
         border-radius: $timeline-image-border-radius;
         cursor: pointer;
     }
+    .mx_MLocationBody_tooltip{
+        z-index: -1 !important;
```

Comment:
Is !important really needed here?

Reply:
Actually, I was trying using important, otherwise, it would take z-index 0. But still, the tooltip is overlapping.
I am not able to figure out the reason.

Reply:
@goelesha do you mean:

1. Your change does not fix the problem?
or
2. You change does fix the problem, but `!important` is needed.

If 2, I would like you to try and find a way to make the match more specific. Maybe add more classes? Do ask if you don't understand.

Reply:
The current is not fixing the problem, although it should. I am not able to figure it out.